gensim/parsing/porter.py

```diff
@@ -329,8 +473,7 @@ def _step4(self):
             self.k = self.j
 
     def _step5(self):
-        """Remove a final -e if _m() > 1, and change -ll to -l if m() > 1.
-        """
+        """Remove a final -e if _m() > 1, and change -ll to -l if m() > 1."""
```

Comment:
Not sure what to put here, but the description "Step 5." would have the same effect. :)